roles/rhel_subscribe/tasks/openshift-enterprise.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Disable all repositories
+  command: subscription-manager repos --disable="*"
+
+- name: Enable RHEL repositories
+  command: subscription-manager repos \
+               --enable="rhel-7-server-rpms" \
+               --enable="rhel-7-server-extras-rpms" \
+               --enable="rhel-7-server-ose-3.1-rpms"
```

Comment:
This will definitely work for now, however we might want to think about having a way to specify the version, since going forward we will not be updating deployment_types along with versions.